index.html

```diff
@@ -10,17 +10,28 @@
 </head>
 <body>
     <header>
-        <h1>All awesome books</h1>
+        <nav class="nav-bar">
+            <p class="logo">Awesome Books</p>
+            <ul class="menu">
+                <li class="menu-item" id="list" >List</li>
```

Comment:
- [ ] As written in the project requirements, Kindly ensure that when any of the links are clicked, the main area changes to display only one of the three content sections. you can add and remove CSS classes to display or hide the content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) add and remove methods.


[Video Demo](https://www.loom.com/share/3e7a93ab5b264fcf8f2c5366723b30bd)